### PR TITLE
New Pull request

### PR DESCRIPTION
**Pull request by rkathir**
### TODO

- [x] get review from someone else
- [ ] actually include some content

@allthedoll your thoughts?

:smile:

If you haven't already, please comment on #3

### CODE REVIEW

File: rkathir_hello_world.py [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello World
```
Comment (rbolinsk823) [inline rkathir_hello_world.py]: ok

Comment (rkathir) [inline rkathir_hello_world.py]: @rkathir is python demo code

**Comment by rbolinsk823** [comment]
looks good

**Comment by rkathir** [comment]
Delete my own branch

**Pull request merged**